api/v1alpha1/storageclassclaim_types.go

```diff
@@ -73,7 +73,9 @@ type StorageClassClaim struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   StorageClassClaimSpec   `json:"spec,omitempty"`
+	// +kubebuilder:validation:Required
```

Comment:
```suggestion
//+kubebuilder:validation:Required
//+kubebuilder:validation:XValidation:rule="self == oldSelf",message="spec is immutable"
```

Remove space between `//` and `+`

Reply:
and switch `self` and `oldself` as doing here https://kubernetes.io/blog/2022/09/29/enforce-immutability-using-cel/#immutablility-after-first-modification

Reply:
done